README.md

```diff
@@ -19,6 +20,13 @@ Read Hacker News from the desktop
  ```
  flatpak install flathub com.github.mdh34.hackup
  ```
+
+### Arch Linux
+Arch Linux users can find Hackup under the name [hackup-git](https://aur.archlinux.org/packages/hackup-git/) in the **AUR**:
+
+`$ aurman -S hackup-git`
```

Comment:
Could you possibly leave this part out? There are lots of AUR helpers avaliable so just leaving the link should be fine :)